mteb/tasks/Reranking/ara/NamaaMrTydiReranking.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+import datasets
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskReranking import AbsTaskReranking
+
+
+class NamaaMrTydiReranking(AbsTaskReranking):
+    metadata = TaskMetadata(
+        name="NamaaMrTydiReranking",
+        description="MrTydi reranking dataset for arabic reranking evaluation",
+        reference="https://huggingface.co/NAMAA-Space",
+        dataset={
+            "path": "NAMAA-Space/mteb-eval-mrtydi",
+            "revision": "bb3638ffe3b2be76fe2e5a4581123923afee5cda",
+        },
+        type="Reranking",
+        category="s2s",
+        modalities=["text"],
+        eval_splits=["test"],
+        eval_langs=["ara-Arab"],
+        main_score="map",
+        date=("2024-12-09", "2024-12-09"),
+        domains=[],
+        task_subtypes=[],
+        license="not specified",
```

Comment:
```suggestion
        license="cc-by-sa-3.0",
```
Assuming from the original dataset